test/e2e/accounts/common.ts

```diff
@@ -27,7 +27,9 @@ export const accountSnapFixtures = (title: string | undefined) => {
   return {
     dapp: true,
     fixtures: new FixtureBuilder()
-      .withPermissionControllerConnectedToTestDapp(false)
+      .withPermissionControllerConnectedToTestDapp({
+        restrictReturnedAccounts: false,
+      })
```

Comment:
this is needed to not break tests, because now we accept passing an account to this method using an object